### PR TITLE
remove map, community, status, translate pages

### PR DESCRIPTION
**Pull request by Sinyu104**
I have removed the map, community, status, translate pages on the navigation bar.
After @Junochiu styling, other pages may be moved to the toolbar.
Feel free to command. 
image

### CODE REVIEW

**Comment by nicolercl** [comment]
Hi Brandy, since the navigation drawer is added to the public feed page, could you also add     `<script src="/js/navigation-loader.js"></script>` to feed.html so that the login/logout and personal page will show up in the navigation drawer on the public feed page as well?

**Comment by Sinyu104** [comment]
> Hi Brandy, since the navigation drawer is added to the public feed page, could you also add `<script src="/js/navigation-loader.js"></script>` to feed.html so that the login/logout and personal page will show up in the navigation drawer on the public feed page as well?

Sure, no problem!

File: src/main/webapp/feed.html [hunk 1]
```diff
@@ -61,10 +64,65 @@
...
+            <div id="content">
+
+                <div id="message-container">Loading...</div>
+            </div>
+        </v-content>
+    </v-app>
 </div>
+<script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/vuetify/dist/vuetify.js"></script>
+<script type="text/javascript" src="/js/navigationbar-loader.js"></script>>
```
Comment (Sinyu104) [inline src/main/webapp/feed.html]: I have already added here.

Comment (nicolercl) [reply src/main/webapp/feed.html]: No that is different, I mean navigation-loader instead of navigationbar-loader.

Comment (Sinyu104) [reply src/main/webapp/feed.html]: Oh okay.
I have added navigation-loader on feed.html page.

**Review by Junochiu — APPROVED** [review]

**Pull request merged**